JSONResourceFile.js

```diff
@@ -141,6 +141,44 @@ JSONResourceFile.prototype.getDefaultSpec = function() {
     return this.defaultSpec;
 };
 
+/**
+ * @private
+ */
+JSONResourceFile.prototype._isPluralData = function(data) {
+    //if (this.project.getProjectType() !== 'webos-dart') return false;
+    if (this.project.options.projectType !== 'webos-dart') return false;
+
+    if (data.indexOf("#" !== -1) && data.indexOf("|") !== -1) {
+        return true;
+    }
+    return false;
+};
+
+/**
+ * @private
+ */
+JSONResourceFile.prototype._parsePluralData = function(data) {
+    var splitData = data.split("|");
+    var parsePlural = {};
+    if (splitData.length > 0) {
+        splitData.forEach(function(item){
+            var categoryMap = {
```

Comment:
Little nitpick... put this variable outside of the method so that it is only defined once, instead of every time this loop runs.

Reply:
I updated it.